Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM node:14
-WORKDIR /usr/src/app
-COPY package.json .
-COPY package-lock.json .
-COPY tsconfig.json .
+FROM node:14-alpine as builder
+RUN apk add --no-cache --virtual .build-deps python make g++
+COPY package.json package-lock.json tsconfig.json entrypoint.sh ./
 COPY src src
-RUN npm ci
-RUN npm run tsc
-RUN mkdir databases
-COPY databases/*.sql databases/
+RUN npm ci && npm run tsc
+
+FROM node:14-alpine as app
+WORKDIR /usr/src/app
+COPY --from=builder node_modules .
+COPY --from=builder dist ./dist
```

Comment:
copy dist only since we're running compiled files, we have no need for the src files